src/include/catalog/catalog_entry/catalog_entry.h

```diff
@@ -30,6 +31,7 @@ class CatalogEntry {
     virtual void serialize(common::Serializer& serializer) const;
     static std::unique_ptr<CatalogEntry> deserialize(common::Deserializer& deserializer);
     virtual std::unique_ptr<CatalogEntry> copy() const = 0;
+    virtual std::string toCypher(main::ClientContext* /*clientContext*/) const { KU_UNREACHABLE; }
```

Comment:
Up to u, but we are slowly moving `copy` to `EXPLICIY_COPY_DEFAULT_MODE(class name)` type of design. The drawback of `copy` is that it can only return `unique_ptr` which maybe sufficient for `CatalogEntry` but not general enough for other classes where an object instance is enough.

If you don't want change this I can make the change afterwards.